compressors/prefix_free_compressors.py

```diff
@@ -134,16 +135,62 @@ def print_tree(self):
     def build_tree(prob_dist) -> PrefixFreeTreeNode:
```

Comment:
maybe just remove this function from `PrefixFreeTree`.. HuffmanTree can still define the function.